src/Serilog.Sinks.Seq/SeqLoggerConfigurationExtensions.cs

```diff
@@ -70,7 +72,7 @@ public static class SeqLoggerConfigurationExtensions
 
             if (bufferBaseFilename == null)
             {
-                sink = new SeqSink(serverUrl, apiKey, batchPostingLimit, defaultedPeriod, eventBodyLimitBytes,
+                sink = new SeqSink(serverUrl, messageHandler, apiKey, batchPostingLimit, defaultedPeriod, eventBodyLimitBytes,
                     controlLevelSwitch);
             }
             else
```

Comment:
The `messageHander` also needs to be passed to `DurableSeqSink()` to be effective when using local file-based buffering.